colabs/artifact_basics/Artifact_Basics.ipynb

```diff
@@ -0,0 +1,355 @@
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/wandb/examples/blob/master/colabs/pytorch/Artifacts_Basics.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>\n",
+        "<!--- @wandbcode{artifacts-basics} -->"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "TbzoV1XvVMug"
+      },
+      "source": [
+        "<img src=\"http://wandb.me/logo-im-png\" width=\"400\" alt=\"Weights & Biases\" />\n",
+        "\n",
+        "<!--- @wandbcode{pytorch-video} -->\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "MGA0PLfsVMuh"
+      },
+      "source": [
+        "\n",
+        "Use [Weights & Biases](https://wandb.com) for machine learning experiment tracking, dataset versioning, and project collaboration.\n",
+        "\n",
+        "<div><img /></div>\n",
+        "\n",
+        "<img src=\"https://wandb.me/mini-diagram\" width=\"650\" alt=\"Weights & Biases\" />\n",
+        "\n",
+        "<div><img /></div>\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "11CLrFgFExMe"
+      },
+      "source": [
+        "\n",
+        "Use W&B Artifacts to track and version data as the inputs and outputs of your W&B Runs. For example, a model training run might take in a dataset as input and trained model as output. In addition to logging hyperparameters and metadata to a run, you can use an artifact to log the dataset used to train the model as input and the resulting model checkpoints as outputs."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "dX5cFz_XCd5h"
+      },
+      "source": [
+        "# Set Up"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "q2EeMdcpC7Dl"
+      },
+      "source": [
+        "In order to log data to our web service, you need to log in and import `wandb`. If this is your first time using W&B, you'll need to sign up for a free account at the link that appears."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "nwEF5d7NChdB"
+      },
+      "outputs": [],
+      "source": [
+        "!pip install wandb\n",
+        "import wandb\n",
+        "wandb.login()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "vp9Zhzp5Hvt1"
+      },
+      "source": [
+        "# Create An Artifact"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "3xEsZzOEnPFY"
+      },
+      "source": [
+        "The general workflow for creating an Artifact is:\n",
+        "\n",
+        "\n",
+        "1.   Intialize a run.\n",
+        "2.   Create an Artifact.\n",
+        "3.   Add a dataset, a model, or another Artifact as an input.\n",
```

Comment:
I find the "as an input" language a little confusing. I suggest just changing it to "Add to it a dataset, model, another artifact or any files or directories you want to track and version"